### DISCUSSION TITLE
This is only for creating new PDFs and not reading in and parsing existing PDFs, right?

### DISCUSSION THREAD

**Discussion by Hypersapien** [QuestPDF/QuestPDF#510] [General]
For instance, you couldn't read in two PDFs and merge them into a single file with this library, right?

**Comment by girlpunk** [comment]
That's correct, yes

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
